### PR TITLE
#45 :bug: update to also clean target in addition to dist

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -597,6 +597,21 @@ normal circumstances.
...
 
+
+#### distDirectory ####
+
+Controls where the clean plugin will delete dist artifacts.  
+
+Default: `${project.basedir}/dist`
+
+
+#### targetirectory ####
```
Comment (peter-mcclonski) [inline README.md]: A: Typo

**Review by peter-mcclonski — APPROVED** [review]

**Review by tylerashcraft — APPROVED** [review]
Looks good.

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CleanHabushuMojo.java [hunk 2]
```diff
@@ -125,11 +131,15 @@ public void execute() throws MojoExecutionException {
...
             filesetsToDelete.add(distArchivesFileset);
+
+            Fileset targetArchivesFileset = createFileset(targetDirectory);
+            filesetsToDelete.add(targetArchivesFileset);
         } catch (IllegalAccessException e) {
             throw new MojoExecutionException("Could not write to private field in Fileset class.", e);
         }
 
         getLog().info(String.format("Deleting distribution archives at %s", distDirectory));
+        getLog().info(String.format("Deleting distribution archives at %s", targetDirectory));
```
Comment (epwilkins) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CleanHabushuMojo.java]: A: "distribution archives" seems like a misnomer

File: README.md [hunk 3]
```diff
@@ -597,6 +597,21 @@ normal circumstances.
 
 Default: `${project.basedir}/target/habushu.placeholder.txt`
 
+
+#### distDirectory ####
```
Comment (epwilkins) [inline README.md:601]: Q/S: It seems like this should be responsible for more than just the clean plugin functionality. Wouldn't we want to use the same property to drive _where_ Habushu generates artifacts, whether distribution or intermediary/target artifacts?

**Review by epwilkins — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (d-ryan-ashcraft) [reply README.md:601]: Yeah - it is.  This was actually already in use for that, I just noticed it wasn't documented and added the documentation here.

File: README.md (hunk 1 above)
Comment (d-ryan-ashcraft) [reply README.md]: Good catch - fixed ✅

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CleanHabushuMojo.java (hunk 2 above)
Comment (d-ryan-ashcraft) [reply habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CleanHabushuMojo.java]: Good catch - fixed ✅

**Pull request merged**